change/@fluentui-react-calendar-compat-32f67f10-9c9e-4876-82cd-dab716bb4f8c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30435&commitId=b910700bd066b231b612d3202854d7a3c5648c6b&env=prod&clientType=FLUENTUI)


  <details><summary><b>DatePicker Compat</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.default.chromium.png|189| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|119| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|140| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|197| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|140| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|200| Changed|
|DatePicker Compat.marked dates.default.chromium.png|189| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|164| Changed|
|DatePicker Compat.when rendering inline, it should not render behind relatively positioned elements.default.chromium.png|189| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|164| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|189| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|197| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|200| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Screenshots are correct, the indicator is gone since we now explicitly create the focus indicator.